deploy/helm-chart/values.yaml

```diff
@@ -49,14 +49,15 @@ connection:
   # must be created before start
   dbName: tsdb
 
-# dataset configuration options. Values presented in this section are defaults. For full list of settings
-# and their default values go to https://github.com/timescale/promscale/blob/master/docs/dataset.md
-datasetConfig:
-  metrics:
-    compress_data: true
-    default_retention_period: 90d
-  traces:
-    default_retention_period: 30d
+# promscale configuration options. Values presented in this section are defaults. For full list of settings
```

Comment:
Should we specify that this takes lower precedence than extraArgs?

Reply:
I agree lets add a comment.

Reply:
Good point :+1: I added a note about it.